guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
The aim of documenting a single golden command that will guarantee the state of our code in every scenario is a worthy one but it feels like we also have to be realistic about it's application. 

I feel that everyone on the team should firstly understand and agree with our intentions, otherwise everyone will carry on with their current methods (which are fine as there are no deployment issues I'm aware of). I for one would continue to trust that anything on master was ready to go (as everything goes via PR) and fetch and pull as normal as that works for me. For these reasons I'm not particularly keen on adding either right now.